src/applications/pensions/config/form.js

```diff
@@ -962,13 +965,37 @@ const formConfig = {
       title: 'Additional information',
       pages: {
         directDeposit: {
-          title: 'Direct deposit',
+          title: 'Direct deposit for Veterans Pension benefits',
           path: 'additional-information/direct-deposit',
           initialData: {},
           uiSchema: {
-            ...titleUI('Direct deposit'),
-            'view:noDirectDeposit': {
-              'ui:title': 'I don’t want to use direct deposit',
+            ...titleUI(
+              'Direct deposit for Veterans Pension benefits',
+              DirectDepositWarning,
+            ),
+            'view:usingDirectDeposit': yesNoUI({
+              title: 'Do you have a bank account to use for direct deposit?',
+            }),
+          },
+          schema: {
+            type: 'object',
+            properties: {
+              'view:usingDirectDeposit': yesNoSchema,
+            },
+            required: ['view:usingDirectDeposit'],
+          },
+        },
+        accountInformation: {
+          title: 'Account information for direct deposit',
+          path: 'additional-information/account-information',
+          initialData: {},
+          depends: usingDirectDeposit,
+          uiSchema: {
+            'view:accountInformationAlert': {
+              ...titleUI(
+                'Account information for direct deposit',
+                AccountInformationAlert,
+              ),
             },
```

Comment:
This view field is actually unnecessary. We don't need to organize or save the title and description to the redux store.

```suggestion
        ...titleUI(
              'Account information for direct deposit',
              AccountInformationAlert,
            ),
```